lib/spack/docs/mirrors.rst

```diff
@@ -266,6 +266,23 @@ is always searched first when attempting to retrieve files for an installation)
 but can be cleared with :ref:`clean <cmd-spack-clean>`; the cache directory can also
 be deleted manually without issue.
 
-Caching includes retrieved tarball archives and source control repositories, but
-only resources with an associated digest or commit ID (e.g. a revision number
-for SVN) will be cached.
+^^^^^^^^^^^^^^^^^^^^^^^
+Version Control Caching
+^^^^^^^^^^^^^^^^^^^^^^^
+
+In general, caching includes retrieved tarball archives and source control
```

Comment:
I would suspect someone might land on these docs and not have context of being part of mirrors - I would add some detail here and open the section like "Spack uses a caching strategy to make retrieval of tarball archives and source repositories more efficient." (or something along those lines) so the first part of the section states exactly the context of the caching and why it's done.

Reply:
Thank you! I have applied your suggestion directly, although I added a bit more clarification to the phrase "source repositories".